common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationActivity.java

```diff
@@ -28,19 +28,27 @@
 import androidx.fragment.app.Fragment;
 
 import com.microsoft.identity.common.internal.ui.DualScreenActivity;
+import com.microsoft.identity.common.java.opentelemetry.SerializableSpanContext;
 import com.microsoft.identity.common.logging.Logger;
 
+import io.opentelemetry.api.trace.SpanContext;
+import lombok.Getter;
+import lombok.experimental.Accessors;
+
 public class AuthorizationActivity extends DualScreenActivity {
 
     public static final String TAG = AuthorizationActivity.class.getSimpleName();
-
+    @Getter
+    @Accessors(prefix = "m")
+    private SpanContext mSpanContext;
     private AuthorizationFragment mFragment;
 
     @Override
     public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         final String methodTag = TAG + ":onCreate";
+        mSpanContext = (SpanContext) getIntent().getExtras().getSerializable(SerializableSpanContext.SERIALIZABLE_SPAN_CONTEXT);
```

Comment:
NPE if getIntent().getExtras() is null?

Reply:
Addressed here: 51e4f91049ddfd1979ec06cc9c4292590937d561